docs/governance/contribute/git-guideline.md

```diff
@@ -0,0 +1,109 @@
+# Git 提交指南
+
+本文介绍了Git提交变更时需要注意的事项，如果拒绝接受本文的内容会导致提交的变更无法被接受。
```

Comment:
Small tip: Chinese and English generally have a small space, you can use this markdown formatting tool: https://github.com/elliotxx/mdfmt